bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/cli/BioStudiesCommandLine.kt

```diff
@@ -0,0 +1,88 @@
+package ac.uk.ebi.biostd.client.cli
+
+import ac.uk.ebi.biostd.client.integration.commons.SubmissionFormat
+import ac.uk.ebi.biostd.client.integration.web.SecurityWebClient
+import org.apache.commons.cli.DefaultParser
+import org.apache.commons.cli.HelpFormatter
+import org.apache.commons.cli.Options
+import org.json.JSONObject
+import org.springframework.web.client.ResourceAccessException
+import org.springframework.web.client.RestClientResponseException
+import java.io.File
+
+const val ATTACHED = "attached"
+const val FORMAT = "format"
+const val INPUT = "input"
+const val PASSWORD = "password"
+const val SERVER = "server"
+const val USER = "user"
+const val CLI_ID = "PT Submit"
+const val FILES_SEPARATOR = ','
+const val NULL_SUBMISSION_ERROR_MSG = "There was a problem performing the submission"
+
+class BioStudiesCommandLine(
+    private val helpFormatter: HelpFormatter = HelpFormatter()
+) {
+    internal val options = Options().apply {
+        addRequiredOption("s", SERVER, true, "BioStudies host url")
+        addRequiredOption("u", USER, true, "User that will perform the submission")
+        addRequiredOption("p", PASSWORD, true, "The user password")
+        addRequiredOption("i", INPUT, true, "Path to the file containing the submission page tab")
+        addRequiredOption("f", FORMAT, true, "Page tab format used to process the given submission")
+        addOption("a", ATTACHED, true, "Comma separated list of paths to the files referenced in the submission")
+    }
```

Comment:
FYI apacha cli does not look bad but I was using `https://github.com/ajalt/clikt` when tested new method and looks pretty nice as we can easily add more commands with different parameters configuration. Error support is also nicer.